node/internal.ts

```diff
@@ -426,7 +427,7 @@ export function _which(tool: string, check?: boolean): string {
 function _tryGetExecutablePath(filePath: string, extensions: string[]): string {
     try {
         // test file exists
-        let stats: fs.Stats = fs.statSync(filePath);
+        let stats: fs.Stats = fs.statSync(_unTildify(filePath));
```

Comment:
This is going to take longer as ultimately it's `let stats: fs.Stats = fs.statSync(filePath);` that is failing, yet in each block the `ENOENT` exception is being eaten, so it passes. Some of these might valid; that's what I have to investigate.  e.g.. https://github.com/Microsoft/vsts-task-lib/blob/master/node/internal.ts#L453 and https://github.com/Microsoft/vsts-task-lib/blob/master/node/internal.ts#L495